CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# How to contribute
+---
+* For this repository
```

Comment:
@Quickz , the `CONTRIBUTING.md` was created in https://github.com/robertoduessmann/health-check/pull/7.

Maybe you can update your branch and make some revisions on created guidelines... 
Like [here](https://github.com/robertoduessmann/health-check/pull/7/files#diff-6a3371457528722a734f3c51d9238c13R57), I think it would be changed to `cd health-check`

In time, thanks for interest in project :blush: